events/KMS/GenerateDataKeyWithoutPlaintext.json

```diff
@@ -9,6 +9,23 @@
     "mitreAttackTechniques": [
         "T1486 - Data Encrypted for Impact"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1485 - Data Destruction",
+            "reason": "The symmetric data key can be used to encrypt or delete critical data, rendering it useless and causing operational disruptions."
+        },
+        {
+            "technique": "T1560 - Archive Collected Data",
+            "reason": "The data key can facilitate the encryption of collected data before exfiltration to avoid detection."
+        },
+        {
+            "technique": "T1496 - Resource Hijacking",
+            "reason": "Attackers could use the symmetric data key to encrypt legitimate resources, thereby disrupting normal operations and potentially leading to financial or operational impacts."
```

Comment:
I believe resource hijacking is about using resources from the company, that have an elevated cost in your favor, like cryptocurrency mining, use of AI, or email services. But in this case, you are not using expensive resources that benefit you.

Reply:
Agreed. I will remove this. I had "Modify Cloud Compute infrastructure" as a potential replacement in mind, but my confidence in that technqiue is low.